recipes/octomap/all/conanfile.py

```diff
@@ -58,6 +65,8 @@ def generate(self):
         tc.variables["BUILD_TESTING"] = False
         if is_msvc(self) and self.options.shared:
             tc.variables["CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS"] = True
+        if Version(self.version) >= "1.10.0":
+            tc.variables["CMAKE_CXX_STANDARD"] = 11
```

Comment:
```suggestion
            tc.variables["CMAKE_CXX_STANDARD"] = self.settings.compiler.get_safe("cppstd", "11").replace("gnu", "")
```

Just a bit safer, as we can use the defined cppstd in settings.